### PR TITLE
Tweak: Test Responsive reverse columns control inoperative [ED-5931]

### PR DESCRIPTION
**Pull request by guyc-E**

### CODE REVIEW

File: tests/playwright/pages/wp-admin-page.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ exports.WpAdminPage = class wpAdminPage {
 		try {
 			await this.page.click( 'text=Create New Page', { timeout: 5000 } );
 		} catch ( err ) {
-			console.error( "Click on Elementor 'Create New Page' button failed" );
+			// console.error( "Click on Elementor 'Create New Page' button failed" );
```
Comment (udidol) [inline tests/playwright/pages/wp-admin-page.js:34]: Don't do this - deal with the problem itself - make sure you are in the dashboard before clicking "Create New Page"

File: tests/playwright/sanity/includes/elements/section/reverse-columns/reverse-columns.test.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+		await reverseColumns.toggle( reverseColumnDevice );
+
+		await expect( firstColumn ).toHaveCSS( 'order', '10' );
+
+		const filteredBreakpoints = Breakpoints.getAll().filter( ( value ) => reverseColumnDevice !== value );
+
+		for ( const breakpoint of filteredBreakpoints ) {
+			await page.click( `#e-responsive-bar-switcher__option-${ breakpoint }` );
+			await expect( firstColumn ).toHaveCSS( 'order', '0' );
+		}
```
Comment (udidol) [inline tests/playwright/sanity/includes/elements/section/reverse-columns/reverse-columns.test.js:110]: Put this into a function that can receive an array of breakpoints, and use this in the "old breakpoints" and "custom breakpoints" tests

File: tests/playwright/sanity/includes/elements/section/reverse-columns/reverse-columns.test.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+	for ( const breakpoint of Breakpoints.getAll() ) {
+		await page.click( `#e-responsive-bar-switcher__option-${ breakpoint }` );
+		if ( 'desktop' === breakpoint ) {
+			await expect( firstColumn ).toHaveCSS( 'order', '0' );
+			continue;
+		}
+		await reverseColumns.toggle( breakpoint );
+		await expect( firstColumn ).toHaveCSS( 'order', '10' );
+    }
```
Comment (udidol) [inline tests/playwright/sanity/includes/elements/section/reverse-columns/reverse-columns.test.js:139]: Put the relevant parts into a generalized function and call this in the Basic breakpoints test and the Experiment breakpoints test.

**Review by udidol — CHANGES_REQUESTED** [review]

File: tests/playwright/pages/wp-admin-page.js (hunk 1 above)
Comment (guyc-E) [reply tests/playwright/pages/wp-admin-page.js:34]: I changed the function completely

**Review by udidol — CHANGES_REQUESTED** [review]

**Review by udidol — APPROVED** [review]

**Pull request merged**